src/pages/index.js

```diff
@@ -200,7 +199,7 @@ function renderCard(data) {
             deleteModal.resetButtonState();
```

Comment:
You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case: `success` or `failure`.